package.json

```diff
@@ -6,10 +6,11 @@
   "main": "src/entrypoint.ts",
   "scripts": {
     "format": "npx prettier -w .",
+    "format:check": "npx prettier --list-different .",
     "lint:types": "tsc",
     "lint": "yarn run lint:types",
     "test": "node test/badges.js && node test/redirects.js",
-    "precommit-redirect": "node build-scripts/sort-redirects.js && git add redirect.json",
+    "precommit-redirect": "node build-scripts/sort-redirects.js && npx prettier -w redirect.json && git add redirect.json",
```

Comment:
Let's fix `sort-redirects.js` to output valid formatted JS instead (add `+ "\n"`)

Reply:
I thought about doing that. The reason I didn't is that if, in the future, some prettier rules were to change such that the output of `sort-redirects.js` was incorrect we'd run into this issue again. It seemed more brittle to just add "\n" then have prettier just run pre-commit.

That said, totally open to just adding `\n` and dropping this line change. Let me know what you want to do.